attestation-agent/attester/Cargo.toml

```diff
@@ -15,9 +15,11 @@ kbs-types.workspace = true
 log.workspace = true
 nix = { version = "0.28", optional = true, features = ["ioctl", "fs"] }
 occlum_dcap = { git = "https://github.com/occlum/occlum", tag = "v0.29.7", optional = true }
+pv = { version = "0.10.0", package = "s390_pv", optional = true }
```

Comment:
is pv_core sufficient maybe, to avoid linking to openssl?

Reply:
Got your real concern.